docs/docker/README.md

```diff
@@ -0,0 +1,57 @@
+## TODO for Dockerization
+* Add maintainer(s)
+  - Maintainer for CR Dockerfile needs to be added after pull request is merged. Ideally the maintainer should be someone from official team with appropriate Docker Hub permission to tag image as `box/clusterrunner:latest`. I can volunteer if needed but that will require collaborator permission for Docker Hub.
```

Comment:
Can you elaborate on what's involved with this? I created a Docker Hub account named boxoss that I can use (box is too short unfortunately). I'm assuming this is something we'd have to do for every new version?